src/utils_Taylor1.jl

```diff
@@ -72,8 +72,8 @@ end
 convert{T<:Number, S<:Number}(::Type{Taylor1{T}}, b::Array{S,1}) =
     Taylor1(convert(Array{T,1},b))
 convert{T<:Number}(::Type{Taylor1{T}}, b::Number) = Taylor1([convert(T,b)], 0)
-convert{T<:Number}(::Type{Taylor1{T}}, a::Taylor1{T}) = a
-convert{T<:Number}(::Type{Taylor1{T}}, b::Array{T,1}) = Taylor1(b)
+# convert{T<:Number}(::Type{Taylor1{T}}, a::Taylor1{T}) = a
```

Comment:
I would suggest just deleting all these methods, instead of commenting them out.